src/openrct2/object/Object.cpp

```diff
@@ -116,12 +116,12 @@ void rct_object_entry::SetName(const std::string_view& value)
     std::memcpy(name, value.data(), std::min(sizeof(name), value.size()));
 }
 
-std::vector<std::string> Object::GetAuthors()
+const std::vector<std::string>& Object::GetAuthors()
```

Comment:
```suggestion
const std::vector<std::string>& Object::GetAuthors() const
```

Make the method const as well, so compiler knows it won't change data